### DISCUSSION TITLE
🧑‍🎓👩‍🎓 Welcome to Community-Website Discussions!

### DISCUSSION THREAD

**Discussion by Kajol-Kumari** [HITK-TECH-Community/Community-Website#480] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  ➡️ Ask questions you’re wondering about :thinking: 
  ➡️ Share ideas. :bulb: 
  ➡️ Engage with other community members :handshake: 
  ➡️ Welcome others and are open-minded. :smile: 

#### Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community. :smile:

**Comment by ashuvssut** [comment]
Hello

Lets use dart-sass instead of node-sass

dart-sass is installed by `npm i sass`

Why do we need to use dart-sass?
dart-sass is offering all new sass updates. sass-lang will no longer be giving updates to node-sass

**for example**: we should use `@use` instead of `@import`. `@use` provides more features. `@import` is going to be deprecated in oct2021

to install **Ref** https://stackoverflow.com/questions/63943756/replace-node-sass-with-dart-sass-in-create-react-app-v3-x

**Reply by Kajol-Kumari** [reply]
Hey @ashuvssut I believe `node-sass` goes well with our requirements and we aren't planing to shift as of now. Btw thanks for the info :)
